recipes/noqt/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0" %}
+
+package:
+  name: noqt
+  version: {{ version }}
+
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - {{ pin_subpackage('noqt4', exact=True) }}
+    - {{ pin_subpackage('noqt5', exact=True) }}
+    - {{ pin_subpackage('noqt6', exact=True) }}
+test:
+  commands:
+    - echo "no tests for this package"
+
+outputs:
+  - name: noqt
+  - name: noqt4
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt4 only ever had a "mega qt" package called qt
+        - qt <0.a0|>=5.a0
+    test:
+      commands:
+        - echo "no tests for this package"
+  - name: noqt5
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt-main is essentially qt5
+        - qt-main <0.a0
+        # before the creation of qt-main, a megapackage called qt existed
+        - qt <0.a0
```

Comment:
should this be loosened to `qt <5.a0` to allow for qt4 to be installed? i feel like that will cause a whole set of other problems.

Reply:
Exactly, hence why I think the distinction between `noqt5` and `noqt4` isn't worth it.